monkey/monkey_island/cc/repositories/mongo_credentials_repository.py

```diff
@@ -85,7 +85,9 @@ def _encrypt_credentials_mapping(self, mapping: Mapping[str, Any]) -> Mapping[st
                 encrypted_component = None
             else:
                 encrypted_component = {
-                    key: self._repository_encryptor.encrypt(value.encode())
+                    key: self._repository_encryptor.encrypt(
+                        b"\0" if value is None else value.encode()
```

Comment:
This might be problematic. If I put a `SSHKeyPair` in the respository with `None` for the public key, I expect it to come out with `None` as well.

Reply:
I did that first but run into some sort of a problem with encryption.

Reply:
Do
```python3
encrypted_component = {
    key: self._repository_encryptor.encrypt(value.encode()) if value else value
    for key, value in credentials_component.items()
}
```